README.md

```diff
@@ -1,19 +1,22 @@
-# reddit_viewer
+# Tabris.js Reddit Viewer Example
 
-## Run
+[![GitPod Logo](images/run-in-gitpod.png)](https://gitpod.io/#https://github.com/eclipsesource/tabris-js-reddit-viewer)
 
-In the project directory, run the [Tabris CLI](https://www.npmjs.com/package/tabris-cli) command:
+## Description
+This is an example Tabris.js project that demonstrates how to use the ListView feature. The app shows Reddit posts, enables them to be filtered and to change the display mode. The subreddits are retrieved from the Reddit API and rendered by ListView.
 
+The project extensively uses `tabris-decorators` extensions such as data binding, dependency injection, ListView, etc. It is installed in addition to the `tabris` module.
+
+## Run
+In the project directory, run the [Tabris CLI](https://www.npmjs.com/package/tabris-cli) command:
 ```
-tabris serve
+npm start
 ```
-
 This will start a JavaScript app code server at a free port and print its URL to the console.
-
-The JavaScript app code can be [side-loaded](https://tabrisjs.com/documentation/2.0/developer-app.html#the-developer-console) in the [developer app](https://tabrisjs.com/documentation/2.0/developer-app.html) if the default config.xml was not changed. Otherwise, the JavaScript app code must be side-loaded in a [debug build](https://tabrisjs.com/documentation/2.0/build.html#building-a-tabrisjs-app) of this app.
+The JavaScript app code can be [side-loaded](https://tabrisjs.com/documentation/3.3/developer-app.html#the-developer-console) in the [developer app](https://tabrisjs.com/documentation/3.3/developer-app.html) if the default config.xml was not changed.
```

Comment:
You should link to the "latest" path instead of "3.3". Remove the config.xml mention here since we can deal with that in the "build" section.